app/src/androidTest/java/ch/sdp/vibester/activity/DownloadActivityTest.kt

```diff
@@ -109,6 +105,42 @@ class DownloadActivityTest {
         }
     }
 
+    /*
+    //Test that takes too long to execute. Uncomment towards the last sprint.
+    @Test
+    fun downloadMultipleSongs() {
```

Comment:
About DownloadActivity testing: I found it excessive to test if the actual song download is correct. I believe other stuff delete/permission/errors are fine to test. Maybe we should separate code that way, so actual download is note tested, but other things are.